airbyte-commons/build.gradle

```diff
@@ -3,7 +3,9 @@ plugins {
 }
 
 dependencies {
-    testImplementation 'org.apache.commons:commons-lang3:3.11'
+    implementation 'org.apache.commons:commons-compress:1.20'
+    implementation 'com.github.docker-java:docker-java:3.2.8'
```

Comment:
Not super fan of bringing docker into every single project. Commons should remain lean

Reply:
(we might need to create a docker-commons or smth like that

Reply:
I will deal with this before we merge this into master